File: tools/addStringsSmallLarge.py
def addStringsSmallLarge(a,b):
    anumb = len(a)
    bnumb = len(b)
    stringc = ""
    if anumb > bnumb:
        stringc = a + b
    elif anumb < bnumb:
        stringc = b + a
    return stringc
print(addStringsSmallLarge("Notebook", "Calculator"))

File: tools/addStringsSmallLarge.js
function addStringsSmallLarge (a, b) {
    var anumb = a.lenght;
    var bnumb = b.lenght;
    var stringc = "";
    if (anumb > bnumb) {
        stringc = a + b
    } else {
        stringc = b + a
    }
    return stringc
}
console.log(
    addStringsSmallLarge("car", "mechanic")
)